frontend/components/landing/LandingCommunityBanner.vue

```diff
@@ -6,34 +6,40 @@
       class="flex justify-center text-lg space-x-12 sm:space-x-8 xl:space-x-12 xl:text-xl"
     >
       <a
-        class="flex items-center space-x-3 hover:text-light-text dark:hover:text-dark-text focus-brand"
+        class="flex items-center space-x-2 hover:text-light-text dark:hover:text-dark-text focus-brand"
```

Comment:
We don't need `flex items-center space-x-2` here as this is taken care of by the component now :)